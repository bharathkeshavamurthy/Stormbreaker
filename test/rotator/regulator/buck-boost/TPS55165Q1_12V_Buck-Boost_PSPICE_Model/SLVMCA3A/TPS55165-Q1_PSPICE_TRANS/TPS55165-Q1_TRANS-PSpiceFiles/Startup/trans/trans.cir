** Profile: "Startup-trans"  [ c:\users\kesha\workspaces\odin\test\rotator\regulator\buck-boost\tps55165q1_12v_buck-boost_pspice_model\slvmca3a\tps55165-q1_pspice_trans\tps55165-q1_trans-pspicefiles\startup\trans.sim ] 

** Creating circuit file "trans.cir" 
** WARNING: THIS AUTOMATICALLY GENERATED FILE MAY BE OVERWRITTEN BY SUBSEQUENT SIMULATIONS

*Libraries: 
* Profile Libraries :
* Local Libraries :
.LIB "../../../tps55165-q1_trans.lib" 
* From [PSPICE NETLIST] section of C:\SPB_Data\cdssetup\OrCAD_PSpiceTIPSpice_Install\17.4.0\PSpice.ini file:
.lib "nom_pspti.lib" 
.lib "nom.lib" 

*Analysis directives: 
.TRAN  0 14m 0 50n 
.OPTIONS ADVCONV
.OPTIONS ABSTOL= 1.0n
.OPTIONS ITL2= 40
.OPTIONS ITL4= 60
.PROBE64 V(alias(*)) I(alias(*)) 
.INC "..\Startup.net" 


.END
