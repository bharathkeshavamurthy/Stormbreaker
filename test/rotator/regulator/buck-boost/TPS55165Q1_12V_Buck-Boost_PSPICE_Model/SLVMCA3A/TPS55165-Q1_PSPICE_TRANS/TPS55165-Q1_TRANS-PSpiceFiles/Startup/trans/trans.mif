 lib=c:\users\kesha\workspaces\odin\test\rotator\regulator\buck-boost\tps55165q1_12v_buck-boost_pspice_model\slvmca3a\tps55165-q1_pspice_trans\TPS55165-Q1_trans.lib, offset=2426, size=54695
 lib=c:\users\kesha\workspaces\odin\test\rotator\regulator\buck-boost\tps55165q1_12v_buck-boost_pspice_model\slvmca3a\tps55165-q1_pspice_trans\TPS55165-Q1_trans.lib, offset=97683, size=492
 lib=c:\users\kesha\workspaces\odin\test\rotator\regulator\buck-boost\tps55165q1_12v_buck-boost_pspice_model\slvmca3a\tps55165-q1_pspice_trans\TPS55165-Q1_trans.lib, offset=98176, size=486
 lib=c:\users\kesha\workspaces\odin\test\rotator\regulator\buck-boost\tps55165q1_12v_buck-boost_pspice_model\slvmca3a\tps55165-q1_pspice_trans\TPS55165-Q1_trans.lib, offset=73468, size=700
 lib=c:\users\kesha\workspaces\odin\test\rotator\regulator\buck-boost\tps55165q1_12v_buck-boost_pspice_model\slvmca3a\tps55165-q1_pspice_trans\TPS55165-Q1_trans.lib, offset=77162, size=700
 lib=c:\users\kesha\workspaces\odin\test\rotator\regulator\buck-boost\tps55165q1_12v_buck-boost_pspice_model\slvmca3a\tps55165-q1_pspice_trans\TPS55165-Q1_trans.lib, offset=57122, size=746
 lib=c:\users\kesha\workspaces\odin\test\rotator\regulator\buck-boost\tps55165q1_12v_buck-boost_pspice_model\slvmca3a\tps55165-q1_pspice_trans\TPS55165-Q1_trans.lib, offset=74169, size=2992
 lib=c:\users\kesha\workspaces\odin\test\rotator\regulator\buck-boost\tps55165q1_12v_buck-boost_pspice_model\slvmca3a\tps55165-q1_pspice_trans\TPS55165-Q1_trans.lib, offset=93581, size=1042
 lib=c:\users\kesha\workspaces\odin\test\rotator\regulator\buck-boost\tps55165q1_12v_buck-boost_pspice_model\slvmca3a\tps55165-q1_pspice_trans\TPS55165-Q1_trans.lib, offset=77863, size=767
 lib=c:\users\kesha\workspaces\odin\test\rotator\regulator\buck-boost\tps55165q1_12v_buck-boost_pspice_model\slvmca3a\tps55165-q1_pspice_trans\TPS55165-Q1_trans.lib, offset=88663, size=764
 lib=c:\users\kesha\workspaces\odin\test\rotator\regulator\buck-boost\tps55165q1_12v_buck-boost_pspice_model\slvmca3a\tps55165-q1_pspice_trans\TPS55165-Q1_trans.lib, offset=57869, size=612
 lib=c:\users\kesha\workspaces\odin\test\rotator\regulator\buck-boost\tps55165q1_12v_buck-boost_pspice_model\slvmca3a\tps55165-q1_pspice_trans\TPS55165-Q1_trans.lib, offset=96710, size=972
 lib=c:\users\kesha\workspaces\odin\test\rotator\regulator\buck-boost\tps55165q1_12v_buck-boost_pspice_model\slvmca3a\tps55165-q1_pspice_trans\TPS55165-Q1_trans.lib, offset=94624, size=771
 lib=c:\users\kesha\workspaces\odin\test\rotator\regulator\buck-boost\tps55165q1_12v_buck-boost_pspice_model\slvmca3a\tps55165-q1_pspice_trans\TPS55165-Q1_trans.lib, offset=83269, size=765
 lib=c:\users\kesha\workspaces\odin\test\rotator\regulator\buck-boost\tps55165q1_12v_buck-boost_pspice_model\slvmca3a\tps55165-q1_pspice_trans\TPS55165-Q1_trans.lib, offset=58482, size=742
 lib=c:\users\kesha\workspaces\odin\test\rotator\regulator\buck-boost\tps55165q1_12v_buck-boost_pspice_model\slvmca3a\tps55165-q1_pspice_trans\TPS55165-Q1_trans.lib, offset=91507, size=1308
 lib=c:\users\kesha\workspaces\odin\test\rotator\regulator\buck-boost\tps55165q1_12v_buck-boost_pspice_model\slvmca3a\tps55165-q1_pspice_trans\TPS55165-Q1_trans.lib, offset=59225, size=742
 lib=c:\users\kesha\workspaces\odin\test\rotator\regulator\buck-boost\tps55165q1_12v_buck-boost_pspice_model\slvmca3a\tps55165-q1_pspice_trans\TPS55165-Q1_trans.lib, offset=80563, size=769
 lib=c:\users\kesha\workspaces\odin\test\rotator\regulator\buck-boost\tps55165q1_12v_buck-boost_pspice_model\slvmca3a\tps55165-q1_pspice_trans\TPS55165-Q1_trans.lib, offset=59968, size=596
 lib=c:\users\kesha\workspaces\odin\test\rotator\regulator\buck-boost\tps55165q1_12v_buck-boost_pspice_model\slvmca3a\tps55165-q1_pspice_trans\TPS55165-Q1_trans.lib, offset=60565, size=596
 lib=c:\users\kesha\workspaces\odin\test\rotator\regulator\buck-boost\tps55165q1_12v_buck-boost_pspice_model\slvmca3a\tps55165-q1_pspice_trans\TPS55165-Q1_trans.lib, offset=92816, size=764
 lib=c:\users\kesha\workspaces\odin\test\rotator\regulator\buck-boost\tps55165q1_12v_buck-boost_pspice_model\slvmca3a\tps55165-q1_pspice_trans\TPS55165-Q1_trans.lib, offset=61162, size=596
 lib=c:\users\kesha\workspaces\odin\test\rotator\regulator\buck-boost\tps55165q1_12v_buck-boost_pspice_model\slvmca3a\tps55165-q1_pspice_trans\TPS55165-Q1_trans.lib, offset=61759, size=726
 lib=c:\users\kesha\workspaces\odin\test\rotator\regulator\buck-boost\tps55165q1_12v_buck-boost_pspice_model\slvmca3a\tps55165-q1_pspice_trans\TPS55165-Q1_trans.lib, offset=62486, size=728
 lib=c:\users\kesha\workspaces\odin\test\rotator\regulator\buck-boost\tps55165q1_12v_buck-boost_pspice_model\slvmca3a\tps55165-q1_pspice_trans\TPS55165-Q1_trans.lib, offset=63215, size=728
 lib=c:\users\kesha\workspaces\odin\test\rotator\regulator\buck-boost\tps55165q1_12v_buck-boost_pspice_model\slvmca3a\tps55165-q1_pspice_trans\TPS55165-Q1_trans.lib, offset=63944, size=736
 lib=c:\users\kesha\workspaces\odin\test\rotator\regulator\buck-boost\tps55165q1_12v_buck-boost_pspice_model\slvmca3a\tps55165-q1_pspice_trans\TPS55165-Q1_trans.lib, offset=90198, size=1308
 lib=c:\users\kesha\workspaces\odin\test\rotator\regulator\buck-boost\tps55165q1_12v_buck-boost_pspice_model\slvmca3a\tps55165-q1_pspice_trans\TPS55165-Q1_trans.lib, offset=64681, size=734
 lib=c:\users\kesha\workspaces\odin\test\rotator\regulator\buck-boost\tps55165q1_12v_buck-boost_pspice_model\slvmca3a\tps55165-q1_pspice_trans\TPS55165-Q1_trans.lib, offset=65416, size=736
 lib=c:\users\kesha\workspaces\odin\test\rotator\regulator\buck-boost\tps55165q1_12v_buck-boost_pspice_model\slvmca3a\tps55165-q1_pspice_trans\TPS55165-Q1_trans.lib, offset=66153, size=736
 lib=c:\users\kesha\workspaces\odin\test\rotator\regulator\buck-boost\tps55165q1_12v_buck-boost_pspice_model\slvmca3a\tps55165-q1_pspice_trans\TPS55165-Q1_trans.lib, offset=66890, size=736
 lib=c:\users\kesha\workspaces\odin\test\rotator\regulator\buck-boost\tps55165q1_12v_buck-boost_pspice_model\slvmca3a\tps55165-q1_pspice_trans\TPS55165-Q1_trans.lib, offset=78631, size=899
 lib=c:\users\kesha\workspaces\odin\test\rotator\regulator\buck-boost\tps55165q1_12v_buck-boost_pspice_model\slvmca3a\tps55165-q1_pspice_trans\TPS55165-Q1_trans.lib, offset=67627, size=734
 lib=c:\users\kesha\workspaces\odin\test\rotator\regulator\buck-boost\tps55165q1_12v_buck-boost_pspice_model\slvmca3a\tps55165-q1_pspice_trans\TPS55165-Q1_trans.lib, offset=68362, size=474
 lib=c:\users\kesha\workspaces\odin\test\rotator\regulator\buck-boost\tps55165q1_12v_buck-boost_pspice_model\slvmca3a\tps55165-q1_pspice_trans\TPS55165-Q1_trans.lib, offset=68837, size=736
 lib=c:\users\kesha\workspaces\odin\test\rotator\regulator\buck-boost\tps55165q1_12v_buck-boost_pspice_model\slvmca3a\tps55165-q1_pspice_trans\TPS55165-Q1_trans.lib, offset=69574, size=604
 lib=c:\users\kesha\workspaces\odin\test\rotator\regulator\buck-boost\tps55165q1_12v_buck-boost_pspice_model\slvmca3a\tps55165-q1_pspice_trans\TPS55165-Q1_trans.lib, offset=70179, size=604
 lib=c:\users\kesha\workspaces\odin\test\rotator\regulator\buck-boost\tps55165q1_12v_buck-boost_pspice_model\slvmca3a\tps55165-q1_pspice_trans\TPS55165-Q1_trans.lib, offset=70784, size=736
 lib=c:\users\kesha\workspaces\odin\test\rotator\regulator\buck-boost\tps55165q1_12v_buck-boost_pspice_model\slvmca3a\tps55165-q1_pspice_trans\TPS55165-Q1_trans.lib, offset=71521, size=736
 lib=c:\users\kesha\workspaces\odin\test\rotator\regulator\buck-boost\tps55165q1_12v_buck-boost_pspice_model\slvmca3a\tps55165-q1_pspice_trans\TPS55165-Q1_trans.lib, offset=72258, size=604
 lib=c:\users\kesha\workspaces\odin\test\rotator\regulator\buck-boost\tps55165q1_12v_buck-boost_pspice_model\slvmca3a\tps55165-q1_pspice_trans\TPS55165-Q1_trans.lib, offset=72863, size=604
