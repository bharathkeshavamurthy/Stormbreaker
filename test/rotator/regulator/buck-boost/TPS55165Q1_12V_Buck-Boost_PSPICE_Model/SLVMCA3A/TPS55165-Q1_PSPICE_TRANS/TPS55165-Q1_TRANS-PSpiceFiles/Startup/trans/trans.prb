[DISPLAYS]
BEGIN DISPLAY LAST SESSION
ANALYSIS TRANSIENT_ANALYSIS
SYMBOL ALWAYS
TRACECOLORSCHEME NORMAL
BEGIN ANAPLOT 1
XBASE
BEGIN XAXIS
XAXISUSERNAME 0 (null)
TEXT Time
RANGEFLAG AUTO
TYPE LINEAR
UNIT s
BEGIN GRIDS
AUTOMATIC
MAJORNUMBERS
MAJORSTYLE NONE
MINORSTYLE NONE
END GRIDS
END XAXIS
BEGIN YAXIS 1
YAXISSIDE LEFT
ACTIVE
RANGEFLAG AUTO
TYPE LINEAR
UNIT V
BEGIN GRIDS
AUTOMATIC
MAJORNUMBERS
MAJORSTYLE LINES
MAJORPATTERN SOLID
MINORSTYLE NONE
END GRIDS
BEGIN TRACE V(VOUT)
MARKERID 11
END TRACE V(VOUT)
BEGIN TRACE V(PG)
MARKERID 12
END TRACE V(PG)
END YAXIS 1
END ANAPLOT 1
BEGIN ANAPLOT 2
BEGIN XAXIS
XAXISUSERNAME 0 (null)
TEXT Time
RANGEFLAG AUTO
TYPE LINEAR
UNIT s
BEGIN GRIDS
AUTOMATIC
MAJORNUMBERS
MAJORSTYLE NONE
MINORSTYLE NONE
END GRIDS
END XAXIS
BEGIN YAXIS 1
YAXISSIDE LEFT
ACTIVE
RANGEFLAG AUTO
TYPE LINEAR
UNIT A
BEGIN GRIDS
AUTOMATIC
MAJORNUMBERS
MAJORSTYLE LINES
MAJORPATTERN SOLID
MINORSTYLE NONE
END GRIDS
BEGIN TRACE I(RLOAD)
MARKERID 13
END TRACE I(RLOAD)
END YAXIS 1
END ANAPLOT 2
BEGIN ANAPLOT 3
BEGIN XAXIS
XAXISUSERNAME 0 (null)
TEXT Time
RANGEFLAG AUTO
TYPE LINEAR
UNIT s
BEGIN GRIDS
AUTOMATIC
MAJORNUMBERS
MAJORSTYLE NONE
MINORSTYLE NONE
END GRIDS
END XAXIS
BEGIN YAXIS 1
YAXISSIDE LEFT
ACTIVE
RANGEFLAG AUTO
TYPE LINEAR
UNIT A
BEGIN GRIDS
AUTOMATIC
MAJORNUMBERS
MAJORSTYLE LINES
MAJORPATTERN SOLID
MINORSTYLE NONE
END GRIDS
BEGIN TRACE I(L1:1)
MARKERID 8
END TRACE I(L1:1)
END YAXIS 1
END ANAPLOT 3
BEGIN ANAPLOT 4
BEGIN XAXIS
XAXISUSERNAME 0 (null)
TEXT Time
RANGEFLAG AUTO
TYPE LINEAR
UNIT s
BEGIN GRIDS
AUTOMATIC
MAJORNUMBERS
MAJORSTYLE NONE
MINORSTYLE NONE
END GRIDS
END XAXIS
BEGIN YAXIS 1
YAXISSIDE LEFT
ACTIVE
RANGEFLAG AUTO
TYPE LINEAR
UNIT V
BEGIN GRIDS
AUTOMATIC
MAJORNUMBERS
MAJORSTYLE LINES
MAJORPATTERN SOLID
MINORSTYLE NONE
END GRIDS
BEGIN TRACE V(L1)
MARKERID 14
END TRACE V(L1)
END YAXIS 1
END ANAPLOT 4
BEGIN ANAPLOT 5
ACTIVE
BEGIN XAXIS
XAXISUSERNAME 0 (null)
TEXT Time
RANGEFLAG AUTO
TYPE LINEAR
UNIT s
BEGIN GRIDS
AUTOMATIC
MAJORNUMBERS
MAJORSTYLE NONE
MINORSTYLE NONE
END GRIDS
END XAXIS
BEGIN YAXIS 1
YAXISSIDE LEFT
ACTIVE
RANGEFLAG AUTO
TYPE LINEAR
UNIT V
BEGIN GRIDS
AUTOMATIC
MAJORNUMBERS
MAJORSTYLE LINES
MAJORPATTERN SOLID
MINORSTYLE NONE
END GRIDS
BEGIN TRACE V(VBAT)
MARKERID 15
END TRACE V(VBAT)
END YAXIS 1
END ANAPLOT 5
END DISPLAY LAST SESSION
BEGIN DISPLAY LAST DISPLAY
ANALYSIS TRANSIENT_ANALYSIS
SYMBOL ALWAYS
TRACECOLORSCHEME NORMAL
BEGIN ANAPLOT 1
ACTIVE
XBASE
BEGIN XAXIS
XAXISUSERNAME 0 (null)
TEXT Time
RANGEFLAG AUTO
TYPE LINEAR
UNIT s
BEGIN GRIDS
AUTOMATIC
MAJORNUMBERS
MAJORSTYLE LINES
MAJORPATTERN SOLID
MINORSTYLE LINES
MINORPATTERN DOT
END GRIDS
END XAXIS
BEGIN YAXIS 1
YAXISSIDE LEFT
ACTIVE
RANGEFLAG NORANGE
TYPE LINEAR
UNIT  
END YAXIS 1
END ANAPLOT 1
END DISPLAY LAST DISPLAY
