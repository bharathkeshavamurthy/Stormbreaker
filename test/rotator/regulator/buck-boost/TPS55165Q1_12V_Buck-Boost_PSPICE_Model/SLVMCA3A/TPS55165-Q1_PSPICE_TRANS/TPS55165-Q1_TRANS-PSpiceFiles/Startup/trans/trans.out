
**** 04/07/21 19:54:35 ******* PSpice 17.4.0 (Nov 2018) ******* ID# 0 ********

 ** Profile: "Startup-trans"  [ c:\users\kesha\workspaces\odin\test\rotator\regulator\buck-boost\tps55165q1_12v_buck-boost_pspice_mod


 ****     CIRCUIT DESCRIPTION


******************************************************************************




** Creating circuit file "trans.cir" 
** WARNING: THIS AUTOMATICALLY GENERATED FILE MAY BE OVERWRITTEN BY SUBSEQUENT SIMULATIONS

*Libraries: 
* Profile Libraries :
* Local Libraries :
.LIB "../../../tps55165-q1_trans.lib" 
* From [PSPICE NETLIST] section of C:\SPB_Data\cdssetup\OrCAD_PSpiceTIPSpice_Install\17.4.0\PSpice.ini file:
.lib "nom_pspti.lib" 
.lib "nom.lib" 

*Analysis directives: 
.TRAN  0 14m 0 50n 
.OPTIONS ADVCONV
.OPTIONS ABSTOL= 1.0n
.OPTIONS ITL2= 40
.OPTIONS ITL4= 60
.PROBE64 V(alias(*)) I(alias(*)) 
.INC "..\Startup.net" 



**** INCLUDING Startup.net ****
* source TPS55165-Q1_TRANS
C_C8         0 VBAT  10u  
X_U1         N167982642 N1679826418 0 0 VBAT 0 N16798810 N16798815 0 PG 0 0 0 0
+  VBAT VBAT VREG VOUT VOUT VREG VREG TPS55165-Q1_TRANS PARAMS: STEADY_STATE=1
X_C15         VOUT 0 CESR PARAMS: C=0.1u ESR=64m IC=5 X=1
C_C1         0 VBAT  0.47u  
R_RLOAD1         VOUT 0  {12/0.7} TC=0,0 
X_C2         VOUT 0 CESR PARAMS: C=10u ESR=2.229m IC=5 X=1
X_C16         VOUT 0 CESR PARAMS: C=10u ESR=2.229m IC=5 X=1
X_C17         VOUT 0 CESR PARAMS: C=10u ESR=2.229m IC=5 X=1
R_R2         PG VOUT  100k TC=0,0 
C_C11         0 VREG  4.7u  
V_VIN         VBAT 0 14
X_U2         N16798810 N167982642 CESR PARAMS: C=100n ESR=29.6m ESL=1N
X_U3         N16798815 N1679826418 CESR PARAMS: C=100n ESR=30m ESL=1N
X_U5         N16798815 N16798810 LDCR PARAMS: L=18m DCR=18m

**** RESUMING trans.cir ****
.END

WARNING(ORPSIM-15256): <X_U2.ESL> not a subcircuit param

WARNING(ORPSIM-15256): <X_U3.ESL> not a subcircuit param

ERROR(ORPSIM-15141): Less than 2 connections at node N167982642.

ERROR(ORPSIM-15141): Less than 2 connections at node N1679826418.

ERROR(ORPSIM-15142): Node X_U2.1 is floating

ERROR(ORPSIM-15142): Node N167982642 is floating

ERROR(ORPSIM-15142): Node X_U3.1 is floating

ERROR(ORPSIM-15142): Node N1679826418 is floating
